src/main/java/reposense/parser/CsvParser.java

```diff
@@ -195,6 +200,20 @@ private String getRowContentAsRawString(final CSVRecord record) {
         return contentAsString;
     }
 
+    /**
+     * Checks if a given header has the right number of columns.
+     * @throws InvalidCsvException if the header does not have as many columns as expected.
+     */
+    private void validateHeader(String[] possibleHeader) throws InvalidCsvException {
```

Comment:
For the javadocs, let's include the input argument in the docs wherever possible.

`Checks if {@code possibleHeader} contains the expected number of columns. ...`

Reply:
Done